packages/lesswrong/lib/collections/postViewTimes/schema.ts

```diff
@@ -0,0 +1,40 @@
+import { foreignKeyField } from "../../utils/schemaUtils";
+
+export const schema: SchemaType<"PostViewTimes"> = {
```

Comment:
I would love some documentation of the fields as well, but I'll leave that to you.